index.html

```diff
@@ -39,8 +39,19 @@
     <link rel="shortcut icon" href="favicon.ico" />
 
     <!-- social media images -->
-    <link rel="image_src" href="/img/hero.png" />
-    <meta property="og:image" content="http://www.brackets.io/img/hero.png" />
+    <!-- Facebook: https://developers.facebook.com/docs/sharing/webmasters#markup -->
+    <meta property="og:url" content="http://brackets.io" />
+    <meta property="og:title" content="A modern, open source code editor that understands web design" />
+    <meta property="og:description" content="Brackets is a lightweight, yet powerful, modern text editor. We blend visual tools into the editor so you get the right amount of help when you want it. With new features and extensions released every 3-4 weeks, it's like getting presents all year long." />
+    <meta property="og:site_name" content="Brackets" />
+    <meta property="og:image" content="http://brackets.io/img/hero@2x.png" />
+    <!-- Twitter Cards: https://dev.twitter.com/cards/overview -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@brackets">
+    <meta name="twitter:creator" content="@brackets">
+    <meta name="twitter:title" content="Brackets - A modern code editor that understands web design">
```

Comment:
Note: We only have 70 chars here. That's why I dropped the `open source` part here.
I'm thinking of dropping the `Brackets` part instead as Twitter actually mentions it often enough.
Thoughts?